pkg/subctl/cmd/join.go

```diff
@@ -301,5 +307,14 @@ func populateSubmarinerSpec(subctlData *datafile.SubctlData) submariner.Submarin
 		Namespace:                SubmarinerNamespace,
 	}
 
+	print("submariner spec is %s", submarinerSpec)
 	return submarinerSpec
 }
+
+func createSAPerCluster(clientSet clientset.Interface, brokerNamespace string) (*v1.Secret) {
+    broker.NewBrokerSA()
+    broker.NewBrokerRole()
```

Comment:
You don't need to create a new role every time, we can have just one role that will be attached to each new SA created for clusters. (via rolebinding)